cmd/skupper/skupper_kube_service.go

```diff
@@ -181,21 +185,26 @@ func (s *SkupperKubeService) BindArgs(cmd *cobra.Command, args []string) error {
 }
 
 func (s *SkupperKubeService) BindFlags(cmd *cobra.Command) {
-	cmd.Flags().BoolVar(&publishNotReadyAddresses, "publish-not-ready-addresses", false, "If specified, skupper will not wait for pods to be ready")
+	cmd.Flags().BoolVar(&bindOptions.PublishNotReadyAddresses, "publish-not-ready-addresses", false, "If specified, skupper will not wait for pods to be ready")
+	cmd.Flags().StringVar(&bindOptions.Namespace, "target-namespace", "", "Expose resources from a specific namespace")
 }
 
 func (s *SkupperKubeService) Unbind(cmd *cobra.Command, args []string) error {
 	silenceCobra(cmd)
 
 	targetType, targetName := parseTargetTypeAndName(args[1:])
 
-	err := s.kube.Cli.ServiceInterfaceUnbind(context.Background(), targetType, targetName, args[0], false)
+	err := s.kube.Cli.ServiceInterfaceUnbind(context.Background(), targetType, targetName, args[0], false, unbindNamespace)
 	if err != nil {
 		return fmt.Errorf("%w", err)
 	}
 	return nil
 }
 
+func (s *SkupperKubeService) UnbindFlags(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&unbindNamespace, "target-namespace", "", "Target namespace for bound service")
```

Comment:
Last comment :)
Just change the description to match what is on UnexposeFlags: `Target namespace for exposed resource`.

Reply:
Done!